app/views/components/GoodsRecordFilterSearchTerm.scala.html

```diff
@@ -0,0 +1,71 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import views.html.components._
+@import models.Country._
+
+
+@this(
+      config: FrontendAppConfig
+)
+
+@(form: Form[_], countries: Seq[Country], page: Int, searchForm: SearchForm)(implicit request: Request[_], messages: Messages)
+
+    @if(!searchForm.searchTerm.isEmpty
+    || !searchForm.countryOfOrigin.isEmpty
+    || !searchForm.statusValue.isEmpty) {
```

Comment:
We can use nonEmpty here instead of having !.isEmpty

```
f(searchForm.searchTerm.nonEmpty
    || searchForm.countryOfOrigin.nonEmpty
    || searchForm.statusValue.nonEmpty)
    ```